core/prelude-core.el

```diff
@@ -394,6 +394,7 @@ Doesn't mess with special buffers."
     "Press <C-Backspace> to kill a line backwards."
     "Press <C-S-Backspace> or <s-k> to kill the whole line."
     "Press <s-j> or <C-^> to join lines."
+    "Press <s-.> or <C-c j> followed by a character to jump to."
```

Comment:
This description is not clear. Maybe "Press ... to jump instantly to the start of a word at any visible window.".  Or something like this...